File: idscp2/src/main/java/de/fhg/aisec/ids/idscp2/idscp_core/Idscp2ConnectionAdapter.java
package de.fhg.aisec.ids.idscp2.idscp_core;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class Idscp2ConnectionAdapter implements Idscp2ConnectionListener {
    private static final Logger LOG = LoggerFactory.getLogger(Idscp2ConnectionAdapter.class);

    @Override
    public void onError(Throwable t) {
        LOG.error("Error received in Idscp2ConnectionAdapter", t);
    }

    @Override
    public void onClose(Idscp2Connection connection) {
    }
}

File: idscp2/src/main/kotlin/de/fhg/aisec/ids/idscp2/idscp_core/Idscp2ConnectionAdapter.kt
package de.fhg.aisec.ids.idscp2.idscp_core

import org.slf4j.LoggerFactory

open class Idscp2ConnectionAdapter : Idscp2ConnectionListener {
    override fun onError(t: Throwable) {
        LOG.error("Error received in Idscp2ConnectionAdapter", t)
    }

    override fun onClose(connection: Idscp2Connection) {}

    companion object {
        private val LOG = LoggerFactory.getLogger(Idscp2ConnectionAdapter::class.java)
    }
}